### ISSUE TITLE
Getting unknown property Request::userIP when running queue

### ISSUE THREAD

**Issue by nickcobley** [verbb/formie#183]
**Description**

On one of our forms, we had the setting enabled to capture the users IP. However when the craft queue is run via Cron to dispatch the email notifications, it appears to be trying to capture the IP at that point. But there is no IP available to capture when running the queue via php cli. 

It fails with the error:
 ```Getting unknown property: craft\console\Request::userIP```

Manually processing the queue then works without error, however the IP captured is my IP not the IP of the form submitter. 

**Steps to reproduce**
1. Create a form with the Capture User IP setting enabled
2. Create an email notification
3. Process the queue using PHP CLI to send the email notification. 

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: No
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.2.23.1
- Craft version: 3.5.13.2
- Multi-site: No

**Comment by nickcobley** [comment]
Followup on this after some more testing. It does appear that the IP has already been captured prior to the email notification being sent, as I can see the correct submission IP in the form submission while the failed queue item is sat there. So it looks like it's making a second attempt to capture the IP when the queue runs.

**Comment by engram-design** [comment]
I think I know the issue. Can you check by changing your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.2.23.1",
  "...": "..."
}
```

Then run `composer update`.

**Comment by nickcobley** [comment]
That's fixed it, thanks Josh.

I'll revert back to the stable version now as I only found it by accident, I don't need to collect IP's.

**Comment by engram-design** [comment]
No problem, appreciate the report!

**Comment by engram-design** [comment]
Fixed in [1.2.24](https://github.com/verbb/formie/releases/tag/1.2.24)
